src/trees/32-byte-code-chunker.md

```diff
@@ -0,0 +1,44 @@
+# 32-byte code-chunker
+
+- [32-byte code-chunker](#32-byte-code-chunker)
+  - [Background reading](#background-reading)
+  - [How does it work?](#how-does-it-work)
+  - [Runtime usage](#runtime-usage)
+  - [Efficiency](#efficiency)
+  - [Implementations](#implementations)
+
+## Background reading
+
+To get a proper background on where this code chunker fits into stateless Ethereum, read the [*Trees*](intro.md) introductory chapter and the [*Code chunking*](data-encoding.md#code-chunking) section of *Data encoding*.
+
+## How does it work?
+
+This approach aims to store code in full 32-byte chunks while efficiently encoding the necessary metadata to validate jump destinations. [This proposal](https://github.com/ipsilon/eof/blob/eof0-dense/spec/eofv0_verkle.md#encode-only-invalid-jumpdests-dense-encoding) contrast with the [31-byte chunker](31-byte-code-chunker.md) by storing metadata separately rather than prepending it to each chunk.
```

Comment:
```suggestion
This approach aims to store code in full 32-byte chunks while efficiently encoding the necessary metadata to validate jump destinations. [This proposal](https://github.com/ipsilon/eof/blob/eof0-dense/spec/eofv0_verkle.md#encode-only-invalid-jumpdests-dense-encoding) contrasts with the [31-byte chunker](31-byte-code-chunker.md) by storing metadata separately rather than prepending it to each chunk.
```